src/activities/drawnumbers/resource/README

```diff
@@ -0,0 +1,5 @@
+
+for drawnumbers images & sound:-
+
+Activity icon :- http://gcompris.net/incoming/lang/lang/words/numbers.png
+Sound source(Button click) :- http://soundbible.com/tags-click.html
```

Comment:
I'm guessing it's for buttonclick.wav, can you put the exact file you have taken (with its license), some of the files listed in this site can't be used.